docs/zh-CN/administrator-guide/variables.md

```diff
@@ -334,3 +334,6 @@ SET forward_to_master = concat('tr', 'u', 'e');
 
     是否将 bitmap 和 hll 类型的 count distinct 查询重写为 bitmap_union_count 和 hll_union_agg 。
 
+* `allow_ignore_order_by`
```

Comment:
maybe allow_ignore_order_by_in_subquery is more better? 
set, inline view, insert can think contain subquery.